go/tasks/pluginmachinery/core/exec_metadata.go

```diff
@@ -11,6 +12,7 @@ import (
 type TaskOverrides interface {
 	GetResources() *v1.ResourceRequirements
 	GetConfig() *v1.ConfigMap
+	GetRetryStrategy() *v1alpha1.RetryStrategy
```

Comment:
Please do not add references to the CRD. These should be references to the core Protobuf objects. This is a cyclic dependency